tutor/controllers/resources.py

```diff
@@ -1,9 +1,15 @@
+from .. import db
 from flask import Flask, Response, render_template, request, redirect, url_for
 from .forms import resourceForm
+from ..models.course import Resource
 
 
-def newResource():
+def newResource(id):
     form = resourceForm()
     if form.validate_on_submit():
-        return redirect(url_for('index_route'))
+       resource= Resource(title=form.title.data, content=form.content.data, course_id=id)
+       db.session.add(resource)
+       db.session.commit() 
+       return redirect(url_for('get_course_info_route', id=id))
     return render_template('resource.html', form=form)
+  
```

Comment:
GitHub complains there is no newline in the end of file.

Reply:
I fixed this now, but since I merged the CI check a few minutes ago, now it indicates that there are many lint fixes to be made in other files of the project (not related to this PR) in order to push the commit .. so I assume I should make all this changes in a new PR and then procceed with this one?

Reply:
Let's merge this PR now and than provide another one which fixes the broken CI. This is an unique example of when it is valuable to merge the PR eventhough the CI broken.